fastlane/Fastfile

```diff
@@ -111,7 +111,7 @@ lane :build_flutter do |options|
     build_type = "--#{build_type}"
   end
 
-  sign_enabled = options[:sign_enabled] || true
+  sign_enabled = options[:sign_enabled] || false
   sign_param = sign_enabled ? '' : '--no-codesign'
```

Comment:
check it

Reply:
Yeah, its confusing, maybe changing it to sign_disabled and set te default to false?  
```suggestion
  sign_disabled = options[:sign_disabled] || false
  sign_param = sign_disabled ? '--no-codesign' : ''
```